xc/xenops_server_xen.ml

```diff
@@ -84,6 +84,8 @@ module VmExtra = struct
 		build_info: Domain.build_info option;
 		ty: Vm.builder_info option;
 		last_start_time: float;
+		nomigrate: bool;  (* platform:nomigrate   at boot time *)
+		nested_virt: bool (* platform:nested_virt at boot time *)
 	} with rpc
```

Comment:
You will need to add a record of defaults below here, and override the `of_rpc` function, e.g.
```
let default_persistent_t = {
    build_info: None;
    ...
}
let persistent_t_of_rpc rpc =
    Rpc.struct_extend rpc (rpc_of_persistent_t default_persistent_t) |> persistent_t_of_rpc
```